src/Core/Framework/Api/Context/Exception/InvalidContextSourceException.php

```diff
@@ -6,6 +6,9 @@
 use Shopware\Core\Framework\Log\Package;
 use Symfony\Component\HttpFoundation\Response;
 
+/**
+ * @deprecated tag:v6.7.0 - will be removed, use UsageDataException::invalidContextSource instead
```

Comment:
there is already this method `\Shopware\Core\Framework\Api\ApiException::invalidAdminSource` so I think, you can use this, and leave this exception class. it is also already extending that domain exception, which probably means, it is caught somewhere

Reply:
deprecation is still there

Reply:
Yes, my bad. was fixed and then reverted by mistake. Thanks. :)